diff --git a/main/gui/source/window/window.cpp b/main/gui/source/window/window.cpp
index 91ca1b1..a95126a 100644
--- a/main/gui/source/window/window.cpp
+++ b/main/gui/source/window/window.cpp
@@ -688,12 +688,10 @@ namespace hex {
     void Window::initGLFW() {
         auto initialWindowProperties = ImHexApi::System::getInitialWindowProperties();
         glfwSetErrorCallback([](int error, const char *desc) {
-            if (error == GLFW_PLATFORM_ERROR) {
-                // Ignore error spam caused by Wayland not supporting moving or resizing
-                // windows or querying their position and size.
-                if (std::string_view(desc).contains("Wayland"))
-                    return;
-            }
+            // Ignore error spam caused by Wayland not supporting moving or resizing
+            // windows or querying their position and size.
+            if (std::string_view(desc).contains("Wayland"))
+                return;
 
             try {
                 log::error("GLFW Error [0x{:05X}] : {}", error, desc);
