From 6eae3ac7c2fbcd4155cd1527f52e6d801decfe80 Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Mon, 7 Oct 2024 11:38:38 +0200
Subject: [PATCH 4/6] Fixups

---
 modules/programs/nix-your-shell.nix | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules/programs/nix-your-shell.nix b/modules/programs/nix-your-shell.nix
index c69c3cf4..6ab78174 100644
--- a/modules/programs/nix-your-shell.nix
+++ b/modules/programs/nix-your-shell.nix
@@ -39,6 +39,10 @@ in {
 
       nushell = mkIf cfg.enableNushellIntegration {
         extraEnv = ''
+          let nix_your_shell_cache = "${config.xdg.cacheHome}/nix-your-shell"
+          if not ($nix_your_shell_cache | path exists) {
+            mkdir $nix_your_shell_cache
+          }
           ${cfg.package}/bin/nix-your-shell nu | save --force ${config.xdg.cacheHome}/nix-your-shell/init.nu
         '';
 
-- 
2.47.0

