From fbd954755733e668643256e30342426604ac7ffa Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Fri, 19 Apr 2024 00:40:03 +0200
Subject: [PATCH 1/3] Expose sources and lib.ctp

---
 modules/home-manager/globals.nix | 13 ++++++++++++-
 1 file changed, 12 insertions(+), 1 deletion(-)

diff --git a/modules/home-manager/globals.nix b/modules/home-manager/globals.nix
index 213ccc1..28811be 100644
--- a/modules/home-manager/globals.nix
+++ b/modules/home-manager/globals.nix
@@ -1,4 +1,4 @@
-{ lib, ... }: {
+{ lib, sources, ... }: {
   options.catppuccin = {
     enable = lib.mkEnableOption "Catppuccin globally";
 
@@ -13,5 +13,16 @@
       default = "mauve";
       description = "Global Catppuccin accent";
     };
+
+    sources = lib.mkOption {
+      type = lib.types.lazyAttrsOf lib.types.raw;
+      readOnly = true;
+      default = sources;
+      description = "Port sources used accross all options";
+    };
+  };
+
+  config.lib = {
+    inherit (lib) ctp;
   };
 }
-- 
2.44.0

