From bbf2600fa45d49311763b4f46be9b20e229b78dc Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Fri, 19 Apr 2024 20:32:04 +0200
Subject: [PATCH 2/2] home-manager(tmux): Add option extraConfig for the plugin

---
 modules/home-manager/tmux.nix | 12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/modules/home-manager/tmux.nix b/modules/home-manager/tmux.nix
index 4cdd2cc..3722020 100644
--- a/modules/home-manager/tmux.nix
+++ b/modules/home-manager/tmux.nix
@@ -18,12 +18,20 @@ let
 in
 {
   options.programs.tmux.catppuccin =
-    lib.ctp.mkCatppuccinOpt "tmux";
+    (lib.ctp.mkCatppuccinOpt "tmux") // {
+      extraConfig = lib.mkOption {
+        type = lib.types.lines;
+        default = "";
+      };
+    };
 
   config.programs.tmux.plugins = lib.mkIf enable [
     {
       inherit plugin;
-      extraConfig = "set -g @catppuccin_flavour '${cfg.flavour}'";
+      extraConfig = lib.concatLines [
+        "set -g @catppuccin_flavour '${cfg.flavour}'"
+        cfg.extraConfig
+      ];
     }
   ];
 }
-- 
2.44.0

