From 6ade578bf817182ec7ba9b81847cb73a616f63d7 Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Mon, 13 May 2024 23:56:30 +0200
Subject: [PATCH] Expose lib.ctp

---
 modules/home-manager/globals.nix | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules/home-manager/globals.nix b/modules/home-manager/globals.nix
index 0f66e18..aae6e4e 100644
--- a/modules/home-manager/globals.nix
+++ b/modules/home-manager/globals.nix
@@ -25,4 +25,8 @@
       description = "Port sources used across all options";
     };
   };
+
+  config.lib = {
+    inherit (lib) ctp;
+  };
 }
-- 
2.45.1

