From 8692c215d01c512a50d6522d1d291b9a9feda50b Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Mon, 13 May 2024 23:56:30 +0200
Subject: [PATCH] Expose lib.ctp

---
 modules/home-manager/globals.nix | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules/home-manager/globals.nix b/modules/home-manager/globals.nix
index ddb54c7..3bbf72b 100644
--- a/modules/home-manager/globals.nix
+++ b/modules/home-manager/globals.nix
@@ -33,4 +33,8 @@
         description = "Port sources used across all options";
       };
   };
+
+  config.lib = {
+    inherit (lib) ctp;
+  };
 }
-- 
2.45.2

