From 493d466705b05a4e6be2ab3b5dcfa69ee1bc9c3f Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Sun, 28 Apr 2024 14:09:40 +0200
Subject: [PATCH] Build fixes

---
 src/Niv/Update.hs | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Niv/Update.hs b/src/Niv/Update.hs
index eeb3ae7..65ccfb8 100644
--- a/src/Niv/Update.hs
+++ b/src/Niv/Update.hs
@@ -10,7 +10,6 @@
 
 module Niv.Update where
 
-import Control.Applicative
 import Control.Arrow
 import qualified Control.Category as Cat
 import Data.Aeson (FromJSON, ToJSON, Value)
-- 
2.44.0

